docs/cards.html

```diff
@@ -0,0 +1,358 @@
+            rerum sapiente? Iste qui expedita delectus sunt quos quasi sint deserunt fugiat.
+            Obcaecati eveniet nemo corrupti aperiam aliquam optio doloremque delectus at architecto.
+        </div>
+        <hr>
+        <div class="px-xs pb-xs d-flex fd-row ai-c">
+            <h4> TrishnaUI 2022 | All Rights Reserved</h4>
+        </div>
+    </div>
+</div>
+<div class="px-sm py-sm">
+<pre>
+    <code class="language-markup" data-prismjs-copy="Copy">
+    <div class="card mx-auto bdr-rad-md">
+        <div class="px-sm pt-xs d-flex fd-col">
+            <h3>John Doe</h4>
+            <h5>Architect & Engineer</h5>    
+        </div>
+        <hr>
+        <div class="px-sm d-flex fd-col" >
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. 
+            Repudiandae quidem est cumque rerum sapiente? Iste qui expedita
+            delectus sunt quos quasi sint deserunt fugiat. Obcaecati eveniet
+            nemo corrupti aperiam aliquam optio doloremque delectus at architecto.
+        </div>
+        <hr>
+        <div class="px-xs pb-xs d-flex fd-row ai-c">
+            <h4> TrishnaUI 2022 | All Rights Reserved</h4>
+        </div>
+    </div>
+    </code>
+</pre>
+</div>
+        
+<div class="px-sm py-sm">
+    <div class="card mx-auto bdr-rad-md">
+        <img src="../images/venice.jpg">
+        <div class="px-sm py-xs d-flex fd-col">
+            <h3>John Doe</h4>
+            <h5>Architect & Engineer</h5>    
+        </div>
+        <div class="px-sm py-sm d-flex fd-col" >
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. 
+        Repudiandae quidem est cumque rerum sapiente? Iste qui 
+        expedita delectus sunt quos quasi sint deserunt 
+        </div>
+        <div class="px-sm py-sm d-flex fd-row ai-c">
+            <h4> TrishnaUI 2022 | All Rights Reserved</h4>
+        </div>
+    </div>
+</div>
+<div class="px-sm py-sm">
+<pre>
+    <code class="language-markup" data-prismjs-copy="Copy">
+        <div class="card mx-auto bdr-rad-md">
+            <img src="../images/venice.jpg">
+            <div class="px-sm py-xs d-flex fd-col">
+                <h3>John Doe</h4>
+                <h5>Architect & Engineer</h5>    
+            </div>
+            <div class="px-sm py-sm d-flex fd-col" >
+            Lorem, ipsum dolor sit amet consectetur adipisicing 
+            elit.Repudiandae quidem est cumque rerum sapiente? 
+            Iste qui expedita delectus sunt quos quasi sint deserunt 
+            </div>
+            <div class="px-sm py-sm d-flex fd-row ai-c">
+                <h4> TrishnaUI 2022 | All Rights Reserved</h4>
+            </div>
+        </div>
+    </code>
+</pre>
+</div>
+
+<div class="px-sm py-sm">
+    <div class="card mx-auto bdr-rad-md">
+        <div class="px-sm py-sm d-flex fd-col">
+            <h3>John Doe</h4>
+            <h5>Architect & Engineer</h5>    
+        </div>
+        <img src="../images/venice.jpg">
```

Comment:
Use alt attribute. If your image does not load, the text inside alt will be shown. Semantically more helpful.